x-pack/elastic-agent/pkg/agent/application/upgrade/step_relink.go

```diff
@@ -21,18 +21,18 @@ func (u *Upgrader) changeSymlink(ctx context.Context, newHash string) error {
 	// create symlink to elastic-agent-{hash}
 	hashedDir := fmt.Sprintf("%s-%s", agentName, newHash)
 
-	agentBakName := agentName + ".bak"
-	symlinkPath := filepath.Join(paths.Config(), agentName)
-	newPath := filepath.Join(paths.Data(), hashedDir, agentName)
+	agentBakName := agentName + ".prev"
```

Comment:
Rename also the var name for consistency?

Reply:
Done